doc/TESTS.rst

```diff
@@ -114,27 +114,41 @@ a test class::
           with pytest.raises(ValueError, match='.*some matching regex.*'):
               ...
 
-Within these test methods, ``assert`` and related functions are used to test
-whether a certain assumption is valid. If the assertion fails, the test fails.
+Within these test methods, the ``assert`` statement or a specialized assertion
+function is used to test whether a certain assumption is valid. If the
+assertion fails, the test fails. Common assertion functions include:
+
+- :func:`numpy.testing.assert_equal` for testing exact elementwise equality
+  between a result array and a reference,
+- :func:`numpy.testing.assert_allclose` for testing near elementwise equality
+  between a result array and a reference (i.e. with specified relative and
+  absolute tolerances), and
+- :func:`numpy.testing.assert_array_less` for testing (strict) elementwise
+  ordering between a result array and a reference.
+
+Note that these assertion functions only compare the numerical vales of the
```

Comment:
```suggestion
Note that these assertion functions only compare the numerical values of the
```

Reply:
We now have `strict=True` on some (`assert_equal`?) to check a bit more (mainly the shape and probably the dtype).

Reply:
Yes, `assert_array_equal`. But it's only that one, I think. How bout we add `strict` to the others and then I can mention it here?

Reply:
That PR got merged, care to update the text too?

Reply:
Sure, I'm happy to update this after adding `strict` to `assert_equal`.

Reply:
Ah missed that there was still another step, merging!